apps/api/v1/pages/api/connected-calendars/_get.ts

```diff
@@ -1,7 +1,7 @@
 import type { NextApiRequest } from "next";
 
 import type { UserWithCalendars } from "@calcom/lib/getConnectedDestinationCalendars";
-import { getConnectedDestinationCalendars } from "@calcom/lib/getConnectedDestinationCalendars";
+import { getConnectedDestinationCalendarsAndEnsureDefaultsInDb } from "@calcom/lib/getConnectedDestinationCalendars";
```

Comment:
Main logic of the fn remains same but the name is changed to reflect that it does DB updates as well